fast/stages/03-gke-multitenant/prod/main.tf

```diff
@@ -17,55 +17,37 @@
 locals {
   labels = merge(var.labels, { environment = var.environment })
 
-  _gke_robot_sa      = "serviceAccount:${module.gke-project-0.service_accounts.robots.container-engine}"
-  _cloud_services_sa = "serviceAccount:${module.gke-project-0.service_accounts.cloud_services}"
-  host_project_bindings = [
-    { role = "roles/container.hostServiceAgentUser", member = local._gke_robot_sa },
+  _gke_robot_sa      = "serviceAccount:${module.gke-project-0.project.service_accounts.robots.container-engine}"
+  _cloud_services_sa = "serviceAccount:${module.gke-project-0.project.service_accounts.cloud_services}"
+  host_project_network_users = [
     { role = "roles/compute.networkUser", member = local._gke_robot_sa },
     { role = "roles/compute.networkUser", member = local._cloud_services_sa }
   ]
 }
 
 module "gke-project-0" {
-  source          = "../../../../modules/project"
-  billing_account = var.billing_account_id
-  name            = "${var.environment}-gke-clusters-0"
-  parent          = var.folder_id
-  prefix          = var.prefix
-  labels          = local.labels
+  source             = "../../../../examples/factories/project-factory"
```

Comment:
it's a bit weird to use an example as a module... if we want to do this, we should have a project-factory module probably

Reply:
Reverted back to project module as discussed